test/common.test.js

```diff
@@ -137,71 +139,89 @@ each([1, 2], function(version) {
+          analytics.called(window.__adroll.record_user, {
             adroll_segments: 'order_created',
             user_id: 'id'
           });
         });
 
         it('should include orderId', function() {
-          analytics.track('Completed Order', { id: 1 });
-          analytics.calledOnce(window.__adroll.record_user, {
+          analytics.track('Completed Order', { orderId: 1 });
+          analytics.called(window.__adroll.record_user, {
             adroll_segments: 'order_created',
             order_id: 1
           });
         });
 
         it('should map revenue for normal track events', function() {
           analytics.track('Teems', { revenue: 17.38 });
-          analytics.calledOnce(window.__adroll.record_user, {
-            adroll_segments: 'Ate habanero cheese',
+          analytics.called(window.__adroll.record_user, {
+            adroll_segments: 'ate_habanero_cheese',
             adroll_conversion_value: 17.38
           });
         });
 
         it('should not map revenue if no revenue is found', function() {
           analytics.track('Teems');
-          analytics.calledOnce(window.__adroll.record_user, {
-            adroll_segments: 'Ate habanero cheese'
+          analytics.called(window.__adroll.record_user, {
+            adroll_segments: 'ate_habanero_cheese'
           });
         });
 
         it('should map revenue to conversion_value', function() {
           analytics.track('Completed Order', { revenue: 1.99 });
-          analytics.calledOnce(window.__adroll.record_user, {
+          analytics.called(window.__adroll.record_user, {
             adroll_segments: 'order_created',
             adroll_conversion_value: 1.99
           });
         });
 
         it('should should send total if no revenue for conversion_value', function() {
           analytics.track('Completed Order', { total: 29.88 });
-          analytics.calledOnce(window.__adroll.record_user, {
+          analytics.called(window.__adroll.record_user, {
             adroll_segments: 'order_created',
-            adroll_conversion_value: 29.88
+            adroll_conversion_value: 29.88,
+            total: 29.88
           });
         });
 
         it('should map revenue as conversion_value and total as custom prop', function() {
           analytics.track('Completed Order', { revenue: 2.99, total: 17.38 });
-          analytics.calledOnce(window.__adroll.record_user, {
+          analytics.called(window.__adroll.record_user, {
             adroll_segments: 'order_created',
             adroll_conversion_value: 2.99,
             total: 17.38
           });
         });
 
         it('should include properties', function() {
-          analytics.track('Completed Order', { revenue: 2.99, id: '12345', sku: '43434-21', other: '1234' });
-          analytics.calledOnce(window.__adroll.record_user, {
+          analytics.track('Completed Order', { revenue: 2.99, orderId: '12345', sku: '43434-21', other: '1234' });
```

Comment:
Nope, since the alias on the `.properties({ orderId: 'adroll_conversion_value' })` function will call call `track.orderId()` as seen [here]((https://github.com/segmentio/facade/blob/112973c96ad9925d9da42b303b84805c5d273f32/lib/track.js#L182) if `properties.orderId` does not exist. And `track.orderId()` actually checks `properties.id()` first before falling back on `properties.orderId()`:

https://github.com/segmentio/facade/blob/112973c96ad9925d9da42b303b84805c5d273f32/lib/track.js#L97

But I figure I'd update the test since our spec for `Completed Order` sends `orderId` at the top level not `id`.